hazelcast-enterprise/Dockerfile

```diff
@@ -35,32 +39,44 @@ ENV HZ_HOME="${HZ_HOME}" \
     CLASSPATH="" \
     JAVA_OPTS=""
 
+LABEL name="Hazelcast IMDG Enterprise" \
+      maintainer="info@hazelcast.com" \
+      vendor="Hazelcast, Inc." \
+      version="8.1" \
+      release="${HZ_VERSION}" \
+      url="http://www.hazelcast.com" \
+      summary="Hazelcast IMDG Enterprise Image, certified to RHEL 8.1" \
+      description="Starts a standalone Hazelcast server instance to form a cluster based on Kubernetes discovery inside Openshift/Kubernetes"
```

Comment:
Could you update this description? This image is not only for OpenShift/Kubernetes.

Reply:
Done 👍